src/participants/create_project.py

```diff
@@ -132,17 +124,17 @@ def handle_wi(self, wid):
         summary = ""
         desc = ""
         if not project:
```

Comment:
did project get renamed prjmeta ?

Reply:
no, project is still the name of the project we are creating